L1Trigger/L1THGCal/interface/backend_emulator/HGCalHistoSeeding_SA.h

```diff
@@ -4,12 +4,13 @@
 #include "L1Trigger/L1THGCal/interface/backend_emulator/HGCalTriggerCell_SA.h"
 #include "L1Trigger/L1THGCal/interface/backend_emulator/HGCalHistogramCell_SA.h"
 #include "L1Trigger/L1THGCal/interface/backend_emulator/HGCalHistoClusteringConfig_SA.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
This include won't be available when running the emulator standalone. Calls to MessageLogger should be done outside the emulator (e.g. in the wrapper).

Reply:
I've removed the `printHistogram` function, it was only used for debugging.